tests/api/security.test.js

```diff
@@ -23,17 +23,27 @@
  */
 
 /* globals sails, User, AccessToken, MachineService, Group */
+/* globals ConfigService, StorageService, Machine, App */
 
 const nano = require('./lib/nanotest');
 
 module.exports = function() {
 
   describe('Security', function() {
-
     let token = null;
     let groupId = null;
+    let storageId = null;
```

Comment:
Each tested portion should be in a dedicated `describe` unit with there own `before` and `after` hook if needed.